specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/openshiftclusters/readme.md

```diff
@@ -26,34 +26,12 @@ These are the global settings for the Azure Red Hat OpenShift API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-11
-```
-
-
-### Tag: package-2023-11
-
-These settings apply only when `--tag=package-2023-11` is specified on the command line.
-
-```yaml $(tag) == 'package-2023-11'
-input-file:
-  - stable/2023-11-22/redhatopenshift.json
-```
-### Tag: package-2023-09-04
-
-These settings apply only when `--tag=package-2023-09-04` is specified on the command line.
-
-``` yaml $(tag) == 'package-2023-09-04'
-input-file:
-  - stable/2023-09-04/redhatopenshift.json
-```
-
-### Tag: package-preview-2023-07
-
-These settings apply only when `--tag=package-preview-2023-07` is specified on the command line.
-
-``` yaml $(tag) == 'package-preview-2023-07'
-input-file:
-  - preview/2023-07-01-preview/redhatopenshift.json
+tag: package-2024-08-12-preview
+suppressions:
+  - code: AvoidAdditionalProperties
```

Comment:
>AvoidAdditionalProperties [](http://example.com/codeflow?start=10&length=25)

[ARMBlockingComment] Please add a where: clause. Also, please give the actual reason this was allowed by the ARM reviewer(s). From comments it sounds like it's due to a "passthru" API. IOW, your API is passing parameters through (unvalidated) to an external API you don't control.